src/_guides/language/language-tour.md

```diff
@@ -2048,6 +2059,12 @@ Assert statements work only in checked mode. They have no effect in
 production mode.
 </div>
 
+{% include checked-mode-2.0.html %}
```

Comment:
This makes it sound like asserts might disappear in 2.0. Could you cover that in Dart 2.0 Updates? (in a later PR is fine)